pwiz_tools/Shared/Common/DataBinding/Controls/QuickFilterForm.cs

```diff
@@ -189,7 +189,11 @@ public virtual void ShowErrorMessage(Exception e)
             }
             else
             {
-                MessageBox.Show(this, e.Message);
+                // If there is no ViewContext (which should never happen), fall back to using the system message box to show the
+                // error message.
+                // Put a line break in the function call so that code inspection passes
+                MessageBox
+                    .Show(this, e.Message);
             }

```

Comment:
There's a mechanism for calling out exceptions to the rule, let me do this one properly before we merge.

Reply:
That's done, I'll assume the various resx changes are good as they are automated and extensive.